Neos.Flow/Classes/Aop/Builder/ProxyClassBuilder.php

```diff
@@ -315,6 +316,9 @@ protected function buildAspectContainer(string $aspectClassName): AspectContaine
     {
         $aspectContainer = new AspectContainer($aspectClassName);
         $methodNames = get_class_methods($aspectClassName);
+        if ($methodNames === null) {
```

Comment:
I guess that's the better place to check this after all. Avoids another `class_exists()` check and also covers potentially other cases.